tools/opensnoop_example.txt

```diff
@@ -148,6 +178,10 @@ optional arguments:
   -d DURATION, --duration DURATION
                         total duration of trace in seconds
   -n NAME, --name NAME  only print process names containing this name
+  -e, --extended_fields
```

Comment:
I think we've used the `--extended-fields` syntax in the past. See [`trace_example.txt`](https://github.com/timdoug/bcc/blob/7a1c85eec5395c020be61f24efbe8c15b464f3bb/tools/trace_example.txt) for instance.